src/main/resources/reference.conf

```diff
@@ -212,6 +206,11 @@ ergo {
 
     # Number of keys to be generated for tests
     # testKeysQty = 5
+
+    keysFilter {
```

Comment:
It'd nice to have comments describing these parameters.

Reply:
done!